build.sbt

```diff
@@ -614,6 +614,7 @@ lazy val scalafixInput = project
     // TODO: I think these are false positives
     unusedCompileDependenciesFilter -= moduleFilter(organization = "org.http4s"),
     scalacOptions -= "-Xfatal-warnings",
+    scalacOptions ~= { _.filterNot(_.startsWith("-Wunused:")) }
```

Comment:
Removes the following options:
```
-Wunused:imports  
-Wunused:patvars  
-Wunused:privates 
-Wunused:locals   
-Wunused:explicits
-Wunused:implicits
-Wunused:nowarn   
-Wunused:params   
-Wunused:linted   
```
These options are defined for Scala 2.13 only.

It turned out that it is not necessary to disable the corresponding options for Scala 2.12 since cross build is not set neither for `scalafixInput` nor for `scalafixOutput` modules.